### ISSUE TITLE
Bug in Linked ZeroMQ Install Script

### ISSUE THREAD

**Issue by dvintel** [edgexfoundry/edgex-go#1835]
# 📚 Docs or Wiki Bug Report

### Description

The README.md points to an install script for ZeroMQ; however, the linked Gist has an error that requires the user to modify the script. It would be great if the link was to a working Gist, rather than something that required manual updates.

### What's the affected URL?**
https://github.com/edgexfoundry/edgex-go/blob/master/README.md#zeromq

### Expected vs Actual Behavior**
Expect the linked script to work, but it is currently broken. 

## 🔥 Exception or Error
when running the script user will see:

`bad interpreter: /usr/bin/bash: no such file or directory`

## Fix:
The Gist can be fixed by changing
`#!/usr/bin/bash`
to
`#!/bin/sh`

**Comment by rsdmike** [comment]
Hey @dvintel That script is only for Ubuntu 16.04 and there is a comment on that gist that echoes the same statement about 18.04 and the change in location of bash. Unfortunately this script is not under the control of EdgeX at this time and isn't something we can update. I think the best course of action would be fork the script to unblock you immediately and make the change. I can bring up having this script be adopted by EdgeX however that may not be immediate.

**Comment by dvintel** [comment]
Cloned and fixed for my current use case, but thought I would bring it up to possibly fix in future.

**Comment by anonymouse64** [comment]
@rsdmike is this still actionable for Fuji? Is it already done?

**Comment by anonymouse64** [comment]
I am moving this to Geneva it sounds like this is just a docs issue and not blocking

**Comment by jluous** [comment]
> I am moving this to Geneva it sounds like this is just a docs issue and not blocking

updated the "setup-zeromq.sh" to fix this issue
